app/main.py

```diff
@@ -0,0 +1,178 @@
+import json
+import os
+
+from typing import List, Optional
+
+import pandas as pd
+import numpy as np
+import uvicorn
+from SBTi.interfaces import PortfolioCompany, ScenarioInterface
+
+from fastapi import FastAPI, File, Form, UploadFile, Body, HTTPException
+from pydantic import BaseModel
+import mimetypes
+import SBTi
+from SBTi.portfolio_aggregation import PortfolioAggregationMethod
+
+app = FastAPI(
+    title="SBTi Finance Temperature Alignment tool API",
+    description="This tool helps companies and financial institutions to assess the temperature alignment of current "
+                "targets, commitments, and investment and lending portfolios, and to use this information to develop "
+                "targets for official validation by the SBTi.",
+    version="0.1.0",
+)
+
+
+mimetypes.init()
+UPLOAD_FOLDER = 'data'
+
+with open('config.json') as f_config:
+    config = json.load(f_config)
```

Comment:
always use path.join for any file loadings (e.g. with `__dirname`). will make it more stable depending on how the code is executed